packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.tsx

```diff
@@ -0,0 +1,296 @@
+                        errors = {
+                            ...errors,
+                            [card_key]: { ...(errors[card_key as TPaymentMethod] ?? {}) },
+                        };
+                        errors[card_key as TPaymentMethod] = {
+                            ...errors[card_key as TPaymentMethod],
+                            [payment_id]: {
+                                ...(errors[card_key as TPaymentMethod]?.[payment_id] ?? {}),
+                                payment_method_identifier: verify_payment_method_identifier,
+                            },
+                        };
+                    } else {
+                        delete errors[card_key as TPaymentMethod]?.[payment_id]?.payment_method_identifier;
+                    }
+                    if (!are_files_uploaded) {
+                        errors = {
+                            ...errors,
+                            [card_key]: { ...(errors[card_key as TPaymentMethod] ?? {}) },
+                        };
+                        errors[card_key as TPaymentMethod] = {
+                            ...errors[card_key as TPaymentMethod],
+                            [payment_id]: {
+                                ...(errors[card_key as TPaymentMethod]?.[payment_id] ?? {}),
+                                files: {},
+                            },
+                        };
+                    }
+                }
+                payment_method_data?.files?.forEach((file, i) => {
+                    if (!file?.name) {
+                        return;
+                    }
+                    const verify_file = isValidFile(file);
+                    if (verify_file) {
+                        errors = {
+                            ...errors,
+                            [card_key]: { ...(errors[card_key as TPaymentMethod] ?? {}) },
+                        };
+                        errors[card_key as TPaymentMethod] = {
+                            ...errors[card_key as TPaymentMethod],
+                            [payment_id]: {
+                                ...(errors[card_key as TPaymentMethod]?.[payment_id] ?? {}),
+                                files: {
+                                    ...errors[card_key as TPaymentMethod]?.[payment_id]?.files,
+                                    [i]: verify_file,
+                                },
+                            },
+                        };
+                    } else {
+                        delete errors?.[card_key as TPaymentMethod]?.[payment_id]?.files?.[i];
+                    }
+                });
+            });
+        });
+        return errors;
+    };
+
+    const handleFormSubmit = async (
+        values: Partial<TProofOfOwnershipFormValue>,
+        action: FormikHelpers<Partial<TProofOfOwnershipFormValue>>
+    ) => {
+        const { setFieldError, setStatus } = action;
+        try {
+            setStatus({ is_btn_loading: true });
```

Comment:
do we really need this status here as we're using this for loading state of the button which I guess can be replaced with `isSubmitting`. Please let me know if I'm missing anything

Reply:
Will make the change